eoscostarica/lifebank
# fix: MapPopup render problem

# Fix MapPopup render problem
This PR provides a fix for the render problem when calling the component
# How does this should be tested?
- `make run`
- Login as sponsor or lifebank and go to the profile section
- Check if there is anything unusual
## Linked issues
closes #423
